Update set-up-snowflake-oauth.md (#6104)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/manage-access/set-up-snowflake-oauth.md

### Locate the redirect URI value

To get started, copy the connection's redirect URI from dbt Cloud:
1. Navigate to **Account settings**.
1. Select **Projects** and choose a project from the list. 
1. Select the connection to view its details and set the **OAuth method** to "Snowflake SSO".
1. Copy the **Redirect URI** for use in the next steps.

<Lightbox src="/img/docs/dbt-cloud/access-control/oauth-connections.jpg" width="70%" title="The OAuth method and Redirect URI inputs for a Snowflake connection in dbt Cloud" />

### Create a security integration

In Snowflake, execute a query to create a security integration. Please find the complete documentation on creating a security integration for custom clients [here](https://docs.snowflake.net/manuals/sql-reference/sql/create-security-integration.html#syntax). 

> In the following `CREATE OR REPLACE SECURITY INTEGRATION` example query, replace `<REDIRECT_URI>` with the Redirect URI (also referred to as the [access URL](/docs/cloud/about-cloud/access-regions-ip-addresses)). You can find this in dbt Cloud as mentioned in the previous [locate the redirect URI value](#locate-the-redirect-uri-value) section.

Author: ```suggestion
In the following `CREATE OR REPLACE SECURITY INTEGRATION` example query, replace `<REDIRECT_URI>` with the Redirect URI (also referred to as the [access URL](/docs/cloud/about-cloud/access-regions-ip-addresses)) you copied in dbt Cloud. To locate the Redirect URI, refer to the previous [locate the redirect URI value](#locate-the-redirect-uri-value) section.
```
